config/locales/en.yml

```diff
@@ -437,15 +438,18 @@ en:
       main_address: Main Address
       other_addresses: Other Addresses
       address: Address
+      mailing_address: Mail
       street: *street
       post_code: *post_code
       city: *city
       kommun: *kommun
       region: *region
       website: Company website
       org_nr: *org_nr
-      delete: "Delete the company"
+      delete: "Delete company"
```

Comment:
Why did you take out "the" ?  Grammatically, I believe it needs it.

If you take it out here, then you should also take it out in `confirm_are_you_sure` to be consistent

Reply:
I removed "the" because the edit button had "Edit company".  I then added "Add address" so as to be consistent.

<img width="441" alt="screen shot 2017-09-30 at 8 24 48 am" src="https://user-images.githubusercontent.com/9968213/31045778-1dad2c20-a5b9-11e7-8ba0-457727994ccd.png">

If we added "the" consistently the buttons above would be very wordy and the view would be more cluttered.

An alert message should be a complete sentence (button / link text is not).